### PR TITLE
Add a header should not be equal to value option

### PR DESCRIPTION
**Pull request by tekante**

### CODE REVIEW

**Comment by sanpii** [comment]
A new feature need a new test. Can you add one?

**Comment by tekante** [comment]
Test added for header not equal to check as requested.

File: src/Context/RestContext.php [hunk 1]
```diff
@@ -113,6 +113,21 @@ public function theHeaderShouldBeEqualTo($name, $value)
     }
 
     /**
+    * Checks, whether the header name is not equal to given text
+    *
+    * @Then the header :name should not equal :value
```
Comment (sanpii) [inline src/Context/RestContext.php]: Missing "to" after "equal".

**Comment by sanpii** [comment]
Merged, thank you.

**Pull request closed without merging**